packages/next/src/client/components/error.tsx

```diff
@@ -24,13 +24,13 @@ const styles: { [k: string]: React.CSSProperties } = {
     fontSize: 24,
     fontWeight: 500,
     verticalAlign: 'top',
-    lineHeight: 49,
+    lineHeight: '49px',
   },
 
   h2: {
     fontSize: 14,
```

Comment:
Should `fontSize` and the other values that used `px` before also be updated?

Reply:
No, the `fontSize` number will append px since it's not unitless number in react.
Found some logic in react https://github.com/facebook/react/blob/01a0c4e12c6aa9732d290e13b1452f72d276934d/packages/react-dom-bindings/src/shared/CSSProperty.js#L11

```js
if (typeof styleValue === 'number') {
        if (
          styleValue !== 0 &&
          !hasOwnProperty.call(isUnitlessNumber, styleName)
        ) {
          valueChunk = stringToChunk(styleValue + 'px'); // Presumes implicit 'px' suffix for unitless numbers
        } else {
          valueChunk = stringToChunk('' + styleValue);
        }
      }
```